packages/tools/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+ - We added functionality so that widget TypeScript typings are prettified on creation.
+
```

Comment:
```suggestion
### Added
 - We added functionality so that widget TypeScript typings are prettified on creation.

### Changed
- We improved the formatting script by preventing prettifying MX test projects.
```